packages/shared/routes/dashboard/settings/views/security/DeleteProfile.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="typescript">
+    import { Button, Text } from 'shared/components'
+    import { localize } from 'shared/lib/i18n'
+    import { openPopup } from 'shared/lib/popup'
+
+    function handleDeleteClick() {
+        openPopup({ type: 'deleteProfile' })
+    }
```

Comment:
Missing the check for non empty profiles:
```suggestion
    function handleDeleteClick() {
        if (get(get(wallet).balanceOverview).balanceRaw > 0) {
            return showAppNotification({
                type: 'error',
                message: locale('error.profile.delete.nonEmptyAccounts'),
            })
        }
        openPopup({ type: 'deleteProfile' })
    }
```

Reply:
Might need the imports too for this suggestion to work

Reply:
updated